src/hipscat_import/index/arguments.py

```diff
@@ -61,6 +63,20 @@ def _check_arguments(self):
         self.input_catalog = Catalog.read_from_hipscat(
             catalog_path=self.input_catalog_path, storage_options=self.input_storage_options
         )
+        if self.include_radec:
+            catalog_info = self.input_catalog.catalog_info
+            self.extra_columns.extend([catalog_info.ra_column, catalog_info.dec_column])
+        if len(self.extra_columns) > 0:
+            # check that they're in the schema
+            schema = self.input_catalog.schema
+            missing_fields = [x for x in self.extra_columns if schema.get_field_index(x) == -1]
+            if len(missing_fields):
+                raise ValueError(f"Some requested columns not in input catalog ({','.join(missing_fields)})")
+        extra_columns = []
+        # Remove duplicates, preserving order
+        # pylint: disable=expression-not-assigned
+        [extra_columns.append(x) for x in self.extra_columns if x not in extra_columns]
```

Comment:
I'm not sure I like using the list comprehension like this to just execute something in the for loop, could the for loop be written out? Means the pylint ignore isn't needed either.